matrix_client/client.py

```diff
@@ -73,6 +74,9 @@ def __init__(self, base_url, token=None, user_id=None,
                 the token) if supplying a token; otherwise, ignored.
             valid_cert_check (bool): Check the homeservers
                 certificate on connections?
+            cache_level (int): One of -1, 0, or 1. 1 fully caches room
```

Comment:
rather than magic -1, 0, 1, might it be nice to have some integer constants defined on the class that the user can refer to?  client.CACHE_NONE or whatever...

Reply:
@ara4n placed the constant as an enum in the module namespace instead of as a
class attribute.

That okay?